openedx/features/content_tagging/rules.py

```diff
@@ -1,24 +1,25 @@
 """Django rules-based permissions for tagging"""
 
+from typing import List
+
 import openedx_tagging.core.tagging.rules as oel_tagging
 import rules
 from django.contrib.auth import get_user_model
 
 from common.djangoapps.student.auth import is_content_creator
+from organizations.models import Organization
 
 from .models import TaxonomyOrg
 
 User = get_user_model()
 
 
-def is_taxonomy_admin(user: User, taxonomy: oel_tagging.Taxonomy = None) -> bool:
+def is_taxonomy_user(user: User, taxonomy: oel_tagging.Taxonomy) -> bool:
     """
-    Returns True if the given user is a Taxonomy Admin for the given content taxonomy.
-
-    Global Taxonomy Admins include global staff and superusers, plus course creators who can create courses for any org.
-    Otherwise, a taxonomy must be provided to determine if the user is a org-level course creator for one of the
-    taxonomy's org owners.
+    Returns True if the given user is a Taxonomy User for the given content taxonomy.
```

Comment:
Please explain more:
```suggestion
    Returns True if the given user is a Taxonomy User for the given content taxonomy.
    
    Taxonomy users include global staff and superusers, plus course creators who can create courses for any org.
    Otherwise, we need a taxonomy provided to determine if the user is an org-level course creator for one of the orgs allowed to use this taxonomy.
```